modules/cnv-connecting-resource-bridge-network.adoc

```diff
@@ -0,0 +1,83 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv-attaching-vm-multiple-networks.adoc
+
+[id="connecting-resource-bridge-network-{context}"]
+= Connecting a resource to a bridge-based network
+
+As a network administrator, you can configure a NetworkAttachmentDefinition
+of type `cnv-bridge` to provide Layer-2 networking to Pods and virtual machines.
+
+.Prerequisites
+
+* {ProductName} 2.0 or newer
+* A Linux bridge must be configured and attached to the correct
+Network Interface Card on every node.
+* If you use VLANs, `vlan_filtering` must be enabled on the bridge.
```

Comment:
we need to add here

`vlan_filtering` must be enabled on the bridge and the NIC should be tagged to all relevant VLANs.

Reply:
@dankenigsberg  I added "* The NIC must be tagged to all relevant VLANs." as another prerequisite in cnv-connecting-resource-bridge-network.adoc. Is that sufficient or do we need to add the actual command "bridge vlan add dev bond0 vid 1-4098 master"?

I also made a couple of other minor changes in https://github.com/openshift/openshift-docs/pull/14351/commits/53f3f7e09d33683882862d62e7d7d9fe55f7644e

One question - do I also need to add additional prerequisites in cnv-pxe-booting-with-mac-address.adoc? 

Let me know if there is anything else I'm missing. I'd like to pass this to QE soon. Thanks. :)

Reply:
@ousleyp "bridge vlan add dev bond0 vid 1-4098 master" is quite an arcane command. Please add it as an "e.g."

Reply:
(I cannot review cnv-pxe-booting-with-mac-address.adoc now)

Reply:
@dankenigsberg thanks! It now reads "For example: `bridge vlan add dev bond0 vid 1-4098 master`"

> (I cannot review cnv-pxe-booting-with-mac-address.adoc now)

No rush. Let me know when you get a chance to review the rest.

Reply:
omg, how come do I see my own typo only when you copy it? VIDs are limited by `4095`. please replace 4098 with 4095.

Reply:
Fixed. :)